### PR TITLE
Disambiguated 500 tokens for the GCI task

### PR DESCRIPTION
**Pull request by abhinav-surendra**

### CODE REVIEW

File: texts/formatted_aroundTheWorld.txt [hunk 1]
```diff
@@ -0,0 +1,1713 @@
...
+	"prpers" prn subj p3 mf pl
+"<are>"
+	"be" vbser pres
+;	"be" vbser pres p1 pl
+;	"be" vbser pres p2 sg
+;	"be" vbser pres p2 pl
+;	"be" vbser pres p3 pl
+"<watched>"
+;	"watch" vblex pp
+	"watch" vblex past
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Should be past participle

File: texts/formatted_aroundTheWorld.txt [hunk 2]
```diff
@@ -0,0 +1,1713 @@
...
+"<by>"
+	"by" adv
+;	"by" pr
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Should be preposition

File: texts/formatted_aroundTheWorld.txt [hunk 3]
```diff
@@ -0,0 +1,1713 @@
...
+	"*Fixx"
+"<,>"
+	"," cm
+"<who>"
+;	"who" prn itg mf sp
+	"who" prn rel an mf sp
+"<has>"
+;	"have" vbhaver pres p3 sg
+	"have" vblex pres p3 sg
+"<been>"
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Should be vbhaver

File: texts/formatted_aroundTheWorld.txt [hunk 4]
```diff
@@ -0,0 +1,1713 @@
...
+	"be" vbser pp
+"<dispatched>"
+;	"dispatch" vblex pp
+	"dispatch" vblex past
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Should be past participle

File: texts/formatted_aroundTheWorld.txt [hunk 5]
```diff
@@ -0,0 +1,1713 @@
...
+	"London" np top sg
+;	"London" np cog sg
+"<in>"
+	"in" adv
+;	"in" pr
+"<search>"
+;	"search" n sg
+;	"search" vblex inf
+	"search" vblex pres
+;	"search" vblex imp
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Should be a noun

File: texts/formatted_aroundTheWorld.txt [hunk 6]
```diff
@@ -0,0 +1,1713 @@
...
+"<he>"
+	"prpers" prn subj p3 m sg
+"<cannot>"
+	"not" adv
+		"can" vbmod pres
+"<secure>"
+;	"secure" adj
+;	"secure" vblex inf
+	"secure" vblex pres
+;	"secure" vblex imp
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Should be infinitive.
Comes after "cannot"

File: texts/formatted_aroundTheWorld.txt [hunk 7]
```diff
@@ -0,0 +1,1713 @@
...
+"<the>"
+	"the" det def sp
+"<Mongolia>"
+	"Mongolia" np top sg
+"<)>"
+	")" rpar
+"<conveying>"
+	"convey" vblex pprs
+;	"convey" vblex ger
+;	"convey" vblex subs
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Should be substitutive verb

File: texts/formatted_aroundTheWorld.txt [hunk 8]
```diff
@@ -0,0 +1,1713 @@
...
+"<that>"
+	"that" cnjsub
+;	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
+"<the>"
+	"the" det def sp
+"<Daily>"
+;	"daily" adj
+;	"daily" adv
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: You commented out both of the options

File: texts/formatted_aroundTheWorld.txt [hunk 9]
```diff
@@ -0,0 +1,1714 @@
...
+	"a" det ind sg
+"<procession>"
+	"procession" n sg
+"<in>"
+;	"in" adv
+	"in" pr
+"<which>"
+;	"which" det itg sp
+;	"which" prn itg mf sp
+;	"which" prn rel an mf sp
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Relative pronoun

File: texts/formatted_aroundTheWorld.txt [hunk 10]
```diff
@@ -0,0 +1,1714 @@
...
+"<is>"
+	"be" vbser pres p3 sg
+"<obviously>"
+	"obviously" adv
+"<not>"
+	"not" adv
+"<going>"
+;	"go" vblex pprs
+	"go" vblex ger
+;	"go" vblex subs
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Present participle

File: texts/formatted_aroundTheWorld.txt [hunk 11]
```diff
@@ -0,0 +1,1714 @@
...
+"<are>"
+	"be" vbser pres
+;	"be" vbser pres p1 pl
+;	"be" vbser pres p2 sg
+;	"be" vbser pres p2 pl
+;	"be" vbser pres p3 pl
+"<lost>"
+	"lost" adj
+;	"lose" vblex past
+;	"lose" vblex pp
```
Comment (dolphingarlic) [inline texts/formatted_aroundTheWorld.txt]: Past participle

**Comment by wei2912** [comment]
@dolphingarlic great thanks for the review!

@abhinav-surendra I'll be conducting a review now.

File: texts/formatted_aroundTheWorld.txt [hunk 12]
```diff
@@ -0,0 +1,1714 @@
...
+"<he>"
+	"prpers" prn subj p3 m sg
+"<missed>"
+;	"miss" vblex pp
+	"miss" vblex past
+"<his>"
+;	"his" adj pos pred
+	"his" det pos sp
+;	"prpers" prn pos p3 m sg
+;	"prpers" det pos p3 m sg
```
Comment (wei2912) [inline texts/formatted_aroundTheWorld.txt]: Pick this instead.

File: texts/formatted_aroundTheWorld.txt [hunk 13]
```diff
@@ -0,0 +1,1714 @@
...
+"<connection>"
+	"connection" n sg
+"<.>"
+	"." sent
+"<He>"
+	"He" np cog sg
+;	"prpers" prn subj p3 m sg
```
Comment (wei2912) [inline texts/formatted_aroundTheWorld.txt]: Pick this instead.

File: texts/formatted_aroundTheWorld.txt [hunk 14]
```diff
@@ -0,0 +1,1714 @@
...
+"<promises>"
+;	"promise" n pl
+	"promise" vblex pres p3 sg
+"<Passepartout>"
+	"*Passepartout"
+"<that>"
+;	"that" cnjsub
+	"that" det dem sg
+;	"that" prn dem mf sg
+;	"that" prn rel an mf sp
```
[the remark this answers is not in the document]
Comment (wei2912) [reply texts/formatted_aroundTheWorld.txt]: Shouldn't this be "cnjsub" instead?

File: texts/formatted_aroundTheWorld.txt [hunk 15]
```diff
@@ -0,0 +1,1714 @@
...
+	"in" pr
+"<Passepartout>"
+	"*Passepartout"
+"<,>"
+	"," cm
+"<who>"
+;	"who" prn itg mf sp
+	"who" prn rel an mf sp
+"<does>"
+	"do" vblex pres p3 sg
```
[the remark this answers is not in the document]
Comment (abhinav-surendra) [reply texts/formatted_aroundTheWorld.txt]: I looked through http://wiki.apertium.org/wiki/Tagging_guidelines_for_English and it does seem like 'vbdo' is the best choice.

File: texts/formatted_aroundTheWorld.txt (hunk 14 above)
[the remark this answers is not in the document]
Comment (abhinav-surendra) [reply texts/formatted_aroundTheWorld.txt]: It does act as a connector, 'cnjsub' makes sense to me as well

**Pull request closed without merging**